### PR TITLE
Fix From conversion from NativeFontHandleWrapper to PathfinderComPtr

### PR DESCRIPTION
**Pull request by fschutt**
See https://github.com/pcwalton/pathfinder/pull/81 and https://github.com/servo/webrender/issues/2645 - this fixes 1 compile error, where we can't convert from a NativeFontHandleWrapper to a ComPtr. We need this, so that PathfinderFontContext::add_native_font() works correctly.

---
This change is [Reviewable](https://reviewable.io/reviews/servo/webrender/2705)

### CODE REVIEW

File: webrender/src/platform/windows/font.rs [hunk 1]
```diff
@@ -439,3 +443,16 @@ impl FontContext {
...
     }
 }
+
+#[cfg(feature = "pathfinder")]
+impl<'a> From<NativeFontHandleWrapper<'a>> for PathfinderComPtr<IDWriteFontFace> {
+    fn from(font_handle: NativeFontHandleWrapper<'a>) -> Self {
+        let system_fc = ::dwrote::FontCollection::system();
+        let font = match system_fc.get_font_from_descriptor(&font_handle.0) {
+            Some(font) => font,
+            None => { panic!("missing descriptor {:?}", font_handle.0) }
```
Comment (pcwalton) [inline webrender/src/platform/windows/font.rs]: nit: no need for {} here

**Comment by fschutt** [comment]
@pcwalton alright, fixed it

**Pull request merged**